snakesist/exceptions.py

```diff
@@ -0,0 +1,23 @@
+class SnakesistError(Exception):
```

Comment:
seems simple enough to me. some thoughts though:

- could `ReadError` and `WriteError` be unified, b/c they would occur in distinct operations?
  - but the separation is helpful w/ expressions like: `Document(url).store(…)`
- should all names be prefixed w/ `Snakesist`, in case my app uses another lib that could yield `ReadError`?

Reply:
thx for the quick feedback

* i'm wondering if providing them with a parent class (something like `OperationError`) might help here.
* good point with the prefixing. i avoided it because the naming feels a bit clunky and i assumed the namespacing might prevent any ambiguity, but i agree that avoiding naming conflicts reliably is definitely more important here. i'll add the prefixes.

Reply:
it might help, but `except (ReadError, WriteError)` is also still bearable imo.

Reply:
ok. then i'd say let's just keep them as they are for now.